spec/resource/application_spec.rb

```diff
@@ -264,11 +264,99 @@
       account.delete if account
     end
 
-    it 'returnes verification email' do
+    it 'returns verification email' do
       expect(verification_emails).to be_kind_of Stormpath::Resource::VerificationEmail
     end
   end
 
+  describe 'create_login_attempt' do
+    let(:account) do
+      directory.accounts.create({
+        email: random_email,
+        given_name: 'Ruby SDK',
+        password: 'P@$$w0rd',
+        surname: 'SDK',
+        username: random_user_name
+      })
+    end
+
+    context 'valid credentials' do
+      let(:username_password_request) do
+        Stormpath::Authentication::UsernamePasswordRequest.new(
+          account.email,
+          "P@$$w0rd"
+        )
+      end
+
+      let(:auth_request) { application.authenticate_account(username_password_request) }
+
+      it 'returns login attempt response' do
+        expect(auth_request).to be_kind_of Stormpath::Authentication::AuthenticationResult
+      end
+
+      it 'containes account data' do
```

Comment:
"contains" instead of "containEs". Please make sure that your IDE has grammar correction because these things are easy to overlook in a code review.